sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/rest_client.hpp

```diff
@@ -618,6 +618,14 @@ namespace Azure { namespace Storage { namespace Blobs {
        * User-defined tags for this blob.
        */
       std::map<std::string, std::string> Tags;
+      /**
+       * A string value that uniquely identifies a blob version.
+       */
+      Nullable<std::string> VersionId;
+      /**
+       * Indicates if this is the current version of the blob.
+       */
+      Nullable<bool> IsCurrentVersion;
```

Comment:
If `VersionId` is not null and `IsCurrentVersion` is null, we should set `IsCurrentVersion` to `false`. Do this at convenience layer.

Reply:
Ok, I get it.